lib/css/modules/_filteReddit.scss

```diff
@@ -135,31 +173,28 @@ body.hideOver18 {
 
 	&-external-filter {
 		line-height: 22px;
-		cursor: pointer;
 		padding: 0 4px;
+		display: flex;
+		cursor: default;
+		align-items: center;
 
-		&:hover {
-			background-color: antiquewhite;
+		.gearIcon {
+			margin: 0 4px 0 1px;
+			cursor: pointer;
 		}
 
-		> a {
+		.toggleButton {
+			transform: scale(.8);
 			position: absolute;
-			width: 100%;
-		}
-
-		> div {
-			margin-left: 20px;
-		}
-
-		&[matches-count]::before {
-			float: right;
-			content: attr(matches-count);
+			right: 0;
 		}
 	}
 
 	&-other {
 		margin: 2px 4px;
 		line-height: 16px;
+		display: grid;
```

Comment:
NTS: grid layout may not work quite right in Edge, and/or may need more Autoprefixer configuration to get it to add `-ms-` prefixes, I'll have to check